CMakeLists.txt

```diff
@@ -1,34 +1,51 @@
-# Copyright 2010, Olivier Stasse, JRL, CNRS/AIST
-
 CMAKE_MINIMUM_REQUIRED(VERSION 2.6)
 
-INCLUDE(cmake/base.cmake)
-INCLUDE(cmake/boost.cmake)
-INCLUDE(cmake/eigen.cmake)
-INCLUDE(cmake/pthread.cmake)
-
+# Setup project properties
 SET(PROJECT_ORG stack-of-tasks)
 SET(PROJECT_NAME dynamic-graph)
 SET(PROJECT_DESCRIPTION "Dynamic graph library")
 SET(PROJECT_URL "http://github.com/${PROJECT_ORG}/${PROJECT_NAME}")
 SET(PROJECT_SUFFIX "-v3")
 
-SET(CUSTOM_HEADER_DIR "${PROJECT_NAME}")
+# Export CMake Target
+SET(PROJECT_USE_CMAKE_EXPORT TRUE)
+
+# Make sure that every header is generated in dynamic-graph
+SET(CUSTOM_HEADER_DIR ${PROJECT_NAME})
+
+# Do not fail on warning
+SET(CXX_DISABLE_WERROR TRUE)
 
+INCLUDE(cmake/base.cmake)
+
+# Specify the project.
+COMPUTE_PROJECT_ARGS(PROJECT_ARGS LANGUAGES CXX)
+PROJECT(${PROJECT_NAME} ${PROJECT_ARGS})
+
+# jrl-cmakemodules handling of usual modules
+INCLUDE(cmake/header.cmake)
```

Comment:
this is already included in base:

https://github.com/jrl-umi3218/jrl-cmakemodules/blob/54ece258eed16da94c7f10979588fcb47b744eb8/base.cmake#L94

Reply:
Removed